qemu-test-rauc-config

```diff
@@ -5,6 +5,10 @@ grubenv=/tmp/boot/grub/grubenv
 # HACK: must be on a separate partition for real systems
 data-directory=/tmp/rauc
 
+[log.testlogger]
+filename=event.log
```

Comment:
Should this be somewhere in `/tmp`? If not, where does this end up?

Reply:
default for relative paths is currently relative to data-directory. Thus `/tmp/rauc/event.log` here.

Reply:
Perhaps a comment here would be nice.